packages/web-components/src/components/va-search-input/va-search-input.tsx

```diff
@@ -384,6 +391,12 @@ export class VaSearchInput {
       .slice(0, 5)
       .sort()
       .map(suggestion => this.formatSuggestion(suggestion));
+
+    if(!this.isTouched) {
+      // only open the listbox after the search input has been touched
+      this.isListboxOpen = false;
+      return
+    }
     this.isListboxOpen = true;
```

Comment:
Just a nit here but would a ternary simplify this logic or would that make it less readable?

```js
this.isTouched ? this.isListBoxOpen = true : this.isListboxOpen = false;
```